### ISSUE TITLE
bundle deployment task doesn't change to current directory

### ISSUE THREAD

**Issue by jeremyolliver** [bundler/bundler#1026]
tldr; deploying with bundler and cap using the git remote_cache strategy (or having any other task change directory before bundle:install) causes bundle config to be not found (and subsequently the gems installed too)

When setting up bundler with capistrano using the included deployment recipe, the task never explicitly changes to either the release or current directory first. I know it specifies the Gemfile location etc, so the task itself runs fine, but it places the bundle config file in the directory it ran from, which means that if it happens to be in another directory (which it is if you use the git remote cache method of capistrano) this means the bundle install works, but subsequent rake tasks (if any) or the web server run from the root can't find the gems installed in shared/bundle without the config file.

Since this is a pretty small change, I might write a patch for this if I've got some spare time soon
